Form1.cs

```diff
@@ -10,11 +10,36 @@
 
 namespace Registros
 {
-    public partial class Form1 : Form
+    public partial class Form_Inicio_Sesion : Form
     {
-        public Form1()
+        public Form_Inicio_Sesion()
         {
             InitializeComponent();
         }
+
+        private void Btniniciosesion_Click(object sender, EventArgs e)
+        {
+            Form_Sesion test = new Form_Sesion();
+            test.ShowDialog();
+            if (test.DialogResult == DialogResult.Yes)
+            {
+
```

Comment:
Verifica si esto debe llevar una operación acá, si es asi y no esta implementada aún: añade un 

//TODO:
```suggestion
if (test.DialogResult == DialogResult.Yes)
    { 
       //TODO: I need to check data base
    }
```

Reply:
**test. ShowDialog ();**

Es el método para llamar o mostrar otro formulario en C#, sin el no obedece la instrucción.

Reply:
corrigo, sin esa instrucción, no se ejecuta el comando solicitado "abrir el formulario requerido"

Reply:
Estoy de acuerdo el condicional, permite validar la respuesta del dialogo si es positivo, para asi proceder con una tarea, pero en este momento esta vacio dentro de el, por tanto, si aun no se piensa implementar algo agregar el TODO, tal como en la sugerencia.